avocado/utils/software_manager.py

```diff
@@ -991,3 +989,7 @@ def install_distro_packages(distro_pkg_map, interactive=False):
 
     elif action == 'show-help':
         parser.print_help()
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
Well the most appropriate solution would be to implement `__main__.py`, but as it's a library inside the same dir as other libraries, this works fine. And if this passes the checkers, we can use it...